src/grandorgue/sound/GOSoundEngine.h

```diff
@@ -81,7 +81,7 @@ class GOSoundEngine {
   GOSoundEngine();
   ~GOSoundEngine();
   void Reset();
-  void Setup(GODefinitionFile *organ_file, unsigned release_count = 1);
+  void Setup(GOOrganController *organ_file, unsigned release_count = 1);
```

Comment:
In most other places you've changed to organ_controller, why not here?

Reply:
Renamed to organControlled